internal/config/view.go

```diff
@@ -0,0 +1,290 @@
+		// If there is one argument in the command i.e. command looks like --> flow config view contract,
+		// Then we print the list of all the present accounts in the configuration.
+		// ELSE IF CONDITION:
+		// If there are arguments == 2 i.e. command looks like --> flow config view contract <contractname>,
+		// Then we print all the details of the contract "<contractname>".
+		// 	If the <contractname> doesn't exist in the configuration, then we print "Contract <contractname> does not exist".
+
+		// Declaring a string variable to print our final list.
+		var list string = ""
+
+		if len(args) == 1 {
+			for _, value := range *state.Contracts() {
+				if count == 0 {
+					list = list + string(value.Name)
+					count = count + 1
+				} else if count > 0 {
+					list = list + ", " + string(value.Name)
+				}
+			}
+			// Return the list of contracts.
+			return &Result{
+				result: fmt.Sprintf("List of Contracts: %s", list),
+			}, nil
+		} else if len(args) == 2 {
+			for _, value := range state.Config().Contracts {
+				if value.Name == args[1] {
+					// Return the properties of a particular contract.
+					return &Result{
+						result: fmt.Sprintf("Contract Name: %s \nSource: %s \nNetwork: %s \nAlias: %s", value.Name, value.Source, value.Network, value.Alias),
+					}, nil
+				}
+			}
+			return &Result{
+				result: fmt.Sprintf("Contract %s does not exist", args[1]),
+			}, nil
+		}
+	}
+
+	// If the argument passed is "emulator".
+	if args[0] == "emulator" {
+
+		// Count variable for printing format.
+		var count int = 0
+
+		// IF CONDITION:
+		// If there is one argument in the command i.e. command looks like --> flow config view emulator,
+		// Then we print the list of all the present emulators in the configuration.
+		// ELSE IF CONDITION:
+		// If there are arguments == 2 i.e. command looks like --> flow config view emulator <emulatorname>,
+		// Then we print all the details of the emulator "<emulatorname>".
+		// 	If the <emulatorname> doesn't exist in the configuration, then we print "Emulator <emulatorname> does not exist".
+
+		// Declaring a string variable to print our final list.
+		var list string = ""
+
+		if len(args) == 1 {
+			for _, value := range state.Config().Emulators {
+				if count == 0 {
+					list = list + string(value.Name)
+					count = count + 1
+				} else if count > 0 {
+					list = list + ", " + string(value.Name)
+				}
+			}
+			// Return the list of emulators.
+			return &Result{
+				result: fmt.Sprintf("List of Emulators: %s", list),
+			}, nil
+		} else if len(args) == 2 {
+			for _, value := range state.Config().Emulators {
+				if value.Name == args[1] {
+					// Return the properties of a particular emulator.
+					return &Result{
+						result: fmt.Sprintf("Emulator Name: %s \nPort: %d \nService Account: %s", value.Name, value.Port, value.ServiceAccount),
+					}, nil
+				}
+			}
+			return &Result{
+				result: fmt.Sprintf("Emulator %s does not exist", args[1]),
+			}, nil
+		}
+	}
```

Comment:
It seems that each of these blocks does the same thing with different values which is a good idea to extract into a function.